scripts/cybershake/install_cybershake.py

```diff
@@ -72,6 +64,13 @@ def main():
     run_name = source
     vel_mod_dir = os.path.join(vm_root_dir, source)
     # print vel_mod_dir
+    valid_vm, message = validate_vm.validate_vm(vel_mod_dir)
+    if not valid_vm:
+        message = "Error: VM {} failed {}".format(source, message)
+        print message
+        error_fp.write(message)
```

Comment:
Use with statements each time? Adds an extra line but makes it clearer in my opinion and limits the time the file is open.